### ISSUE TITLE
Overscale in ECDIS - Key governing principles

### ISSUE THREAD

**Issue by alvarosanuy** [iho-ohi/S-101-Documentation-and-FC#94]
**Refer to paper S-101PT11-08.7.** The 'Scales subWG' was tasked to:

 - [ ] 1. Discuss and agree on the key governing principles that are to drive Overscale performance in ECDIS
 - [ ] 2. Prepare changes to  relevant documentation as required (S-98 Annex C and S-101 PS)
 - [ ] 3. Submit change proposals to the S-101PT Chair before S100WG8 (13 - 17 November 2023)
 - [ ] 4. Submit final proposal to the S-164 subWG to prepare new Tests
 - [ ] 5. Submit final proposal to the focus group created at ENCWG8 to draft an 'S101 Educational Document' (to include an Overscale section)

Please review the proposal as presented by Australia in paper S-101PT11-08.7 (below) and **provide your comments by Wednesday 01 November 2023**

**A. Route Monitoring:**

- A section of an ECDIS display is considered Overscaled when the corresponding Data Coverage/s forming the screen image, at that location, are displayed at a MSVS that is larger than their maxDS.
- The position of the ship drives the calculation of the Overscale Indication factor. If a Data Coverage, different to the one the ship is navigating on, is displayed Overscaled, the Overscale Indication must not be shown (unless a ‘mouse hover’ function is implemented by OEM).
- Overscale Indication is to be shown when MSVS > maxDS.
- Overscale Indication is calculated as follows: maxDS /MSVS and expressed as a factor in the following format: xn.d (i.e. x1.7).
- The Data Coverage containing the position of the ship must never be covered by the Overscale Pattern.
- Overscale Pattern is to be shown on any Data Coverage feature forming the ECDIS display whenever MSVS => 2x maxDS (see exception above).
- The boundary between Data Coverages having a jump in scale of more than 3 maxDS steps (the ‘steps’ are taken from S-101 PS Table 3.1) must be depicted using the “Scale boundary’ line symbology. When the boundary between Data Coverage is also the boundary between datasets, linestyle SCLBDY51 is to be used instead.
- When scale boundaries of smaller scale Data Coverage areas overlap larger scale Data Coverage areas, that portion of the scale boundary which intersects the larger scale Data Coverage area must not be visible.

**B. Route Planning:**

- Behave as per Route Monitoring using the centre of the ECDIS display window as if it was the position (Lat/Long) of the ship

**Comment by alvarosanuy** [comment]
> * The boundary between Data Coverages having a jump in scale of more than 3 maxDS steps (the ‘steps’ are taken from S-101 PS Table 3.1) must be depicted using the “Scale boundary’ line symbology. When the boundary between Data Coverage is also the boundary between datasets, linestyle SCLBDY51 is to be used instead.

Does anyone know what is this 'Scale Boundary' trying to achieve? What benefit does it suppose to bring to the mariner??

**Comment by DavidGrant-NIWC** [comment]
It's supposed to show where the navigational purpose of the data changes. Doesn't (currently) make much sense in S-101.

![image](https://github.com/iho-ohi/S-101-Documentation-and-FC/assets/62906211/8d10a053-f982-4c47-99db-bebb02221e24)

**Comment by JeffWootton** [comment]
@alvarosanuy and @DavidGrant-NIWC : 

Should this be a discussion within the S-98 Sub-Group rather than the S-101PT? It seems to me that this is more a generic ECDIS Issue rather than specific to S-101?

**Comment by alvarosanuy** [comment]
> Should this be a discussion within the S-98 Sub-Group rather than the S-101PT? It seems to me that this is more a generic ECDIS Issue rather than specific to S-101?

Yeas - The recomendations need revieweing as ODS was introduced thous - @kusala69

**Comment by kusala9** [comment]
yes, we started on this section yesterday. Agreed that it will be combined in S-98 with the loading strategy pieces as well. I said we will probably need a more dedicated meeting to try and come up with new words for these sections which I will do in the next couple of weeks. We can discuss benefits (or otherwise) then.

**Comment by JeffWootton** [comment]
Discussed at a dedicated VTC amongst relevant Subject Matter Experts. Agreement reached as to the intention of the modelling of the scales in S-101 as included in Edition 2.0.0 and the way that these are supposed to work in ECDIS.

Close Issue pending full testing and feedback.
